android/src/main/java/com/oney/WebRTCModule/PermissionUtils.java

```diff
@@ -0,0 +1,133 @@
+package com.oney.WebRTCModule;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.os.ResultReceiver;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.content.ContextCompat;
+
+
+/**
+ * Helper module for dealing with dynamic permissions, introduced in Android M (API level 23).
+ */
+public class PermissionUtils {
+    /**
+     * Constants for internal fields in the <tt>Bundle</tt> exchanged between the activity
+     * requesting the permissions and the auxiliary activity we spawn for this purpose.
+     */
+    private static final String GRANT_RESULT = "GRANT_RESULT";
+    private static final String PERMISSION = "PERMISSION";
+    private static final String REQUEST_CODE = "REQUEST_CODE";
+    private static final String RESULT_RECEIVER = "RESULT_RECEIVER";
+
+    /**
+     * Incrementing counter for permissions requests. Each request must have a unique numeric code.
+     */
+    private static int requestCode;
+
+    /**
+     * Requests the given permission. The callback will we called with the requested permission and
+     * the granted result. See: https://developer.android.com/reference/android/content/pm/PackageManager.html#PERMISSION_GRANTED
+     * for the result constants.
+     *
+     * @param context Application context / activity.
+     * @param permission Permission which the application is interested in requesting.
+     * @param callback Callback where the results will be reported.
+     */
+    public static void requestPermission(Context context,
+                                         String permission,
```

Comment:
The code does accommodate for that, though I designed that helper to handle them one by by one for simplicity in the implementation. This was deliberate.